ludwig/typing.py

```diff
@@ -0,0 +1,28 @@
+"""Common typing for Ludwig dictionary parameters."""
+from typing import Any, Dict
+
+# Dictionary of parameters used to configure an input or output feature.
+# https://ludwig.ai/latest/configuration/features/supported_data_types/
+FeatureConfigDict = Dict[str, Any]
```

Comment:
On our call I recommended using @PublicAPI annotations, however these type aliases are basically global variables (not classes or functions) so annotations won't work here.  Recommend just adding `# Public API` to the module-level doc string instead.

Reply:
Done.